Volume/Models/Organization.swift

```diff
@@ -72,7 +74,36 @@ extension Organization {
+let capsu = Organization(backgroundImageURL: "", bio: "", bioShort: "", contentType: .cultural, id: "21", name: "Cornell Asian Pacific Student Union", profileImageURL: "", rssName: "", rssURL: "", slug: "capsu", websiteURL: "")
+let pantsimprov = Organization(backgroundImageURL: "", bio: "", bioShort: "", contentType: .comedy, id: "22", name: "Pants Improv Comedy", profileImageURL: "", rssName: "", rssURL: "", slug: "pantsimprov", websiteURL: "")
+let whistlingshrimp = Organization(backgroundImageURL: "", bio: "", bioShort: "", contentType: .comedy, id: "23", name: "Whistling Shrimp", profileImageURL: "", rssName: "", rssURL: "", slug: "whistlingshrimp", websiteURL: "")
+let midnightcomedy = Organization(backgroundImageURL: "", bio: "", bioShort: "", contentType: .comedy, id: "24", name: "Midnight Comedy", profileImageURL: "", rssName: "", rssURL: "", slug: "midnightcomedy", websiteURL: "")
+let thrift = Organization(backgroundImageURL: "", bio: "", bioShort: "", contentType: .awareness, id: "25", name: "Cornell Thrift", profileImageURL: "", rssName: "", rssURL: "", slug: "thrift", websiteURL: "")
+let aaiv = Organization(backgroundImageURL: "", bio: "", bioShort: "", contentType: .spiritual, id: "26", name: "Asian American Intervarsity", profileImageURL: "", rssName: "", rssURL: "", slug: "aaiv", websiteURL: "")
+let cornellchimes = Organization(backgroundImageURL: "", bio: "", bioShort: "", contentType: .music, id: "27", name: "Cornell Chimes", profileImageURL: "", rssName: "", rssURL: "", slug: "cornellchimes", websiteURL: "")
+let shimtah = Organization(backgroundImageURL: "", bio: "", bioShort: "", contentType: .music, id: "28", name: "Shimtah", profileImageURL: "", rssName: "", rssURL: "", slug: "shimtah", websiteURL: "")
+let yamatai = Organization(backgroundImageURL: "", bio: "", bioShort: "", contentType: .music, id: "29", name: "Yamatai", profileImageURL: "", rssName: "", rssURL: "", slug: "yamatai", websiteURL: "")
+let mediumdesigncollective = Organization(backgroundImageURL: "", bio: "", bioShort: "", contentType: .art, id: "30", name: "Medium Design Collective", profileImageURL: "", rssName: "", rssURL: "", slug: "mediumdesigncollective", websiteURL: "")
+let dti = Organization(backgroundImageURL: "", bio: "", bioShort: "", contentType: .academic, id: "31", name: "Design & Tech Initiative", profileImageURL: "", rssName: "", rssURL: "", slug: "dti", websiteURL: "")
+let appdev = Organization(backgroundImageURL: "", bio: "", bioShort: "", contentType: .academic, id: "32", name: "AppDev", profileImageURL: "", rssName: "", rssURL: "", slug: "appdev", websiteURL: "")
+let bhangra = Organization(backgroundImageURL: "", bio: "", bioShort: "", contentType: .dance, id: "33", name: "Bhangra", profileImageURL: "", rssName: "", rssURL: "", slug: "bhangra", websiteURL: "")
```

Comment:
again, there are a lot of fields that are empty for all these organizations. they are probably not causing any issues right now but consider taking them out or utilizing them in the future